### ISSUE TITLE
Customizable Chat Windows - Feature Request

### ISSUE THREAD

**Issue by aclark13** [ClassicUO/ClassicUO#205]
Ability to customize what text shows up in a certain text window.
1. Allow multiple chat windows/journals
2. Allow user to select chat type (ie Combat, System, Guild, Party, Staff, etc)
3. Allow users to set hues, fonts, transparency, etc to each window.

*Idea is akin to Shadowbane's handling of chat windows and could be tweaked to fit UO, but i think the base function would be great!*

Video of Shadowbane Chat
https://streamable.com/662us

**Comment by natryl** [comment]
4. Allow users to enter and exit chat mode. Hitting enter will disable hotkeys and bring up chat menu interface for communication. Hitting enter again will exit and re-enable hotkeys

**Comment by zolter** [comment]
### (4) - done.
but it not disabled hotkeys

![40SslYp](https://user-images.githubusercontent.com/1370602/55666298-8ea2a900-5855-11e9-8305-51f1aeec445f.jpg)
![40SsmJm](https://user-images.githubusercontent.com/1370602/55666299-8ea2a900-5855-11e9-8d15-424ceb902086.jpg)

**Comment by aclark13** [comment]
https://streamable.com/662us

**Comment by oroechimaru** [comment]
from a duplicate request i posted:

often the journal is filled with system message spam which leads you to miss out on conversations that quickly leave the log. can either of these exist?

a. longer journal (2x-3x).
b. a filter to just show chats/conversation

often you miss what someone says and by the time u try finding it in the journal its gone due to system message spam

thanks

**Comment by BlackTeaOutlands** [comment]
Issue : "Additional buttons also activate chat" - not all of them work for me. \ / | doesn't work (guild, party, alliance). Only one that works is ! = yell.

**Comment by aclark13** [comment]
> 
> 
> Issue : "Additional buttons also activate chat" - not all of them work for me. \ / | doesn't work (guild, party, alliance). Only one that works is ! = yell.

Something must be wrong on your end then because I have tested all of them, and they do indeed work.
![chat](https://user-images.githubusercontent.com/38568913/61000021-2666d600-a322-11e9-9f49-ded25ca044bf.gif)

**Comment by oroechimaru** [comment]
black tea did you update to latest dev build yet via appveyor or powershell? if not please test latest builds before updating the posts. thanks.

**Comment by BlackTeaOutlands** [comment]
Yes, I am on the latest build. To be clear, I have "activate chat after pressing Enter" enabled. Those other keys, except ! = Yell, do not work unless I have first activated the chat by pressing Enter.

**Comment by oroechimaru** [comment]
Please allow a longer in game journal. It would be handy if cuo/razor/system message could be filtered or moved to their own tab.

**Comment by MKletz** [comment]
Having the chat box movable/tabbed like World of Warcraft would be great. Tabs for Public/Guild/Party
